app/src/main/res/values/strings.xml

```diff
@@ -517,4 +516,82 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
+    <string name="quest_buildingType_detached">Detached house</string>
+    <string name="quest_buildingType_detached_description">free-standing house</string>
+    <string name="quest_buildingType_terrace">Terraced house</string>
+    <string name="quest_buildingType_terrace_description">row of residential dwellings, each with its own entrance, also known as row-house or townhouse</string>
+    <string name="quest_buildingType_hotel">Hotel</string>
+    <string name="quest_buildingType_dormitory">Dormitory</string>
+    <string name="quest_buildingType_houseboat">Houseboat</string>
+    <string name="quest_buildingType_bungalow">Bungalow</string>
+    <string name="quest_buildingType_bungalow_description">single-storey flat house, holiday cottage</string>
+    <string name="quest_buildingType_static_caravan">Static caravan</string>
+
+    <string name="quest_buildingType_commercial">Commercial</string>
+    <string name="quest_buildingType_commercial_description">building where general commercial activities take place</string>
+    <string name="quest_buildingType_industrial">Industrial</string>
+    <string name="quest_buildingType_industrial_description">building where industrial process takes place, e.g. factory</string>
+    <string name="quest_buildingType_retail">Shop</string>
+    <string name="quest_buildingType_warehouse">Warehouse</string>
+    <string name="quest_buildingType_warehouse_description">building for storage of goods</string>
+    <string name="quest_buildingType_kiosk">Kiosk</string>
+
+    <string name="quest_buildingType_religious">Religious</string>
+    <string name="quest_buildingType_church">Church</string>
+    <string name="quest_buildingType_chapel">Chapel</string>
+    <string name="quest_buildingType_cathedral">Cathedral</string>
+    <string name="quest_buildingType_mosque">Mosque</string>
+    <string name="quest_buildingType_temple">Temple</string>
+    <string name="quest_buildingType_synagogue">Synagogue</string>
+
+    <string name="quest_buildingType_civic">Civic building</string>
+    <string name="quest_buildingType_school">School</string>
+    <string name="quest_buildingType_college">College</string>
+    <string name="quest_buildingType_hospital">Hospital</string>
+    <string name="quest_buildingType_public">Public building</string>
+    <string name="quest_buildingType_public_description">accessible to general public, e.g. town hall or court house</string>
+    <string name="quest_buildingType_stadium">Stadium</string>
+    <string name="quest_buildingType_train_station">Train station</string>
+    <string name="quest_buildingType_transportation">Public transportation building</string>
+    <string name="quest_buildingType_university">University</string>
+
+    <string name="quest_buildingType_cars">For cars</string>
+    <string name="quest_buildingType_carport">Carport</string>
```

Comment:
At least for me carport in not clear at all. "roof covering road vehicles"? "garage without walls"?

Reply:
[Yes.](https://en.wikipedia.org/wiki/Carport)

Reply:
I added now the following description:
"covered structure used to protect vehicles from rain and snow, mostly open on two or more sides"
Is it better understandable now?

Reply:
Yes, thanks!